src/app/combo/cascading-combos/cascading-combos.component.ts

```diff
@@ -18,8 +18,8 @@ export class CascadingCombos {
 
     @ViewChild("inputGroupCountry", { read: IgxInputGroupComponent}) public inputGroupCountry: IgxInputGroupComponent;
     @ViewChild("inputGroupProvince", { read: IgxInputGroupComponent}) public inputGroupProvince: IgxInputGroupComponent;
-    @ViewChild("inputCountry") public inputCountry: HTMLInputElement;
-    @ViewChild("inputProvince") public inputProvince: HTMLInputElement;
+    // @ViewChild("inputCountry") public inputCountry: HTMLInputElement;
```

Comment:
Do you need this commented code?

Reply:
Yes, I have logged separate [issue](https://github.com/IgniteUI/igniteui-angular-samples/issues/770) for that changes to @IvayloG and @ViktorSlavov in order not to delay this PR